models/cooperation.js

```diff
@@ -47,6 +47,11 @@ const cooperationSchema = new Schema(
         required: [true, FIELD_MUST_BE_SELECTED('receiver role')]
       }
     },
+    title: {

```

Comment:
I think title should be required and added to test

Reply:
done